### ISSUE TITLE
Choosing content negotiation form selection will make config that causes API crash

### ISSUE THREAD

**Issue by dstockto** [zfcampus/zf-apigility-admin#277]
If you choose an option from the Content Negotiation tab and save, the resulting module.config.php will result in an error that causes a stack trace in the API.

![image](https://cloud.githubusercontent.com/assets/775393/7195617/ead1a2e2-e47f-11e4-9160-6153cc9ca48e.png)

The resulting change in the module.config.php:

![image](https://cloud.githubusercontent.com/assets/775393/7195623/141285ae-e480-11e4-8186-4a08d8ad60f4.png)

And the resulting crash:

![image](https://cloud.githubusercontent.com/assets/775393/7195627/3353e5c0-e480-11e4-839e-36308489a096.png)

**Comment by codeaid** [comment]
Can confirm the same is happening here. A diff for `module/Api/config/module.config.php`:

```diff
     'zf-content-negotiation' => [
         'controllers' => [
-            'Api\\V1\\Rpc\\MyService\\Controller' => 'Json',
+            'Api\\V1\\Rpc\\MyService\\Controller' => [
+                'content_name' => 'HalJson',
+                'selectors' => [
+                    'ZF\\Hal\\View\\HalJsonModel' => [
+                        0 => 'application/json',
+                        1 => 'application/*+json',
+                    ],
+                ],
+            ],
         ],
         'accept_whitelist' => [
```

**Comment by toringe** [comment]
Same here. From a fresh install of 1.1.0.

**Comment by yo-han** [comment]
Yup, same here.

**Comment by weierophinney** [comment]
Fixed in zfcampus/zf-apigility-admin-ui#40 to release with 1.1.1 this week.
